util/conversion/conversion.go

```diff
@@ -138,28 +137,66 @@ func GetFuzzer(scheme *runtime.Scheme, funcs ...fuzzer.FuzzerFuncs) *fuzz.Fuzzer
 
 // FuzzTestFunc returns a new testing function to be used in tests to make sure conversions between
 // the Hub version of an object and an older version aren't lossy.
-func FuzzTestFunc(scheme *runtime.Scheme, hub conversion.Hub, dst conversion.Convertible, funcs ...fuzzer.FuzzerFuncs) func(*testing.T) {
+func FuzzTestFunc(scheme *runtime.Scheme, hub conversion.Hub, dst conversion.Convertible, beforeCompare func(c1, c2 conversion.Convertible),
```

Comment:
If we want to avoid a breaking signature change in release-0.3, we could use a package-level variable to store `beforeCompare` funcs, and let unit tests register them before calling this function.